derive_builder_core/src/builder.rs

```diff
@@ -345,9 +345,9 @@ macro_rules! default_builder {
             enabled: true,
             ident: syn::Ident::new("FooBuilder", ::proc_macro2::Span::call_site()),
             pattern: Default::default(),
-            derives: &vec![],
-            struct_attrs: &vec![],
-            impl_attrs: &vec![],
+            derives: &[],
+            struct_attrs: &[],
+            impl_attrs: &[],
```

Comment:
If you're going to include these in the PR, you'll need to do a `rebase -i` to clean up the commits before merge (I wouldn't want to squash this into the bulk of the change, but I would like to have the meat of the change be a single commit)

Reply:
Ah, I meant to ask about those rather than committing. The [useless_vec lint](https://rust-lang.github.io/rust-clippy/rust-1.68.0/index.html#useless_vec), in conjunction with
```rust
#![deny(warnings, missing_docs)]
```
was preventing me from compiling.

Reply:
rebased this out.